addons/website/views/website_views.xml

```diff
@@ -174,10 +175,25 @@
                     <field name="create_uid" invisible="1"/>
                     <field name="write_uid"/>
                     <field name="write_date"/>
+                    <field name="is_tracked"/>
                 </tree>
             </field>
         </record>
 
+        <record id="website_pages_view_search" model="ir.ui.view">
+            <field name="name">website.page.view.search</field>
+            <field name="model">website.page</field>
+            <field name="arch" type="xml">
+                <search string="Website Pages" >
+                    <filter string="Published" name="published" domain="[('website_published', '=', True)]"/>
```

Comment:
Simple suggestion: maybe add the "published" filter to the default action context?
So the user lands on published page?

Or maybe not, let me know what you think.